src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useCallback, useEffect, useMemo } from "react";
+import { useParams } from "react-router";
+
+import { AuditoriumVenue } from "types/venues";
+
+import { translateIndex } from "utils/auditorium";
+
+import { useAuditoriumSection } from "hooks/auditoriumSections";
+import { useVenueId } from "hooks/useVenueId";
+
+import UserProfilePicture from "components/molecules/UserProfilePicture";
+
+import { Video } from "../Video";
+
+import {
+  DEFAULT_COLUMNS_NUMBER,
+  DEFAULT_ROWS_NUMBER,
+  SEAT_SIZE,
+  SEAT_SIZE_MIN,
+  SEAT_SPACING,
+  VIDEO_MIN_WIDTH_IN_SEATS,
+} from "./constants";
+
+import "./Section.scss";
+
+export interface SectionProps {
+  venue: AuditoriumVenue;
+}
+
+export const Section: React.FC<SectionProps> = ({ venue }) => {
+  const {
+    iframeUrl,
+    rows = DEFAULT_ROWS_NUMBER,
+    columns = DEFAULT_COLUMNS_NUMBER,
+  } = venue;
+
+  const { sectionId } = useParams<{ sectionId: string }>();
```

Comment:
Yes, we can, since the Route path for this component looks like this:
   
```
  <Route path={`${match.path}/section/:sectionId`}>
        <Section venue={venue} />
      </Route>
```

We can't be sure that it will be a valid id. But this case is handled inside the hook `useAuditoriumSection`

Reply:
But since that route is external,m to this component, there’s nothing stopping someone rendering the component from elsewhere, or changing that route and not updating the component.

For the minimal ‘cost’ of making this optional and explicitly checking it, since it’s a dynamic parameter, I feel it’s safer ‘defensive coding’ that may avoid future issues.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1251/commits/75ae7353c6a7e5d30d83b9f7309d8c755ee318c1